stored_queries/djornl/djornl_fetch_clusters.yaml

```diff
@@ -0,0 +1,49 @@
+name: djornl_fetch_clusters
+description: Fetch all nodes that are members of the specified cluster(s), and the edges and nodes within the specified distance (number of hops) of those nodes.
+params:
+  type: object
+  properties:
+    cluster_i2_ids:
+      title: Cluster I2 IDs
+      description: Cluster I2 IDs to locate
+      items: {type: integer}
+      default: []
+      examples: [[1], [3, 5]]
+    cluster_i4_ids:
+      title: Cluster I4 IDs
+      description: Cluster I4 IDs to locate
+      items: {type: integer}
+      examples: [[2], [4, 6]]
+      default: []
+    cluster_i6_ids:
+      title: Cluster I6 IDs
+      description: Cluster I6 IDs to locate
+      items: {type: integer}
+      examples: [[666], [999, 333]]
+      default: []
```

Comment:
This is a pretty ugly way to do things -- I am still waiting to hear from the Jacobson lab about how they want to use cluster data in terms of searching, etc., but I wanted to be able to search for more than one cluster at a time. I think the best way to represent this data will probably be to have a single `clusters` field in the `node` document, and then have cluster membership listed in the form `<cluster_name>-<cluster_id>`. I will wait and see what they say first, though.